managed/src/test/java/com/yugabyte/yw/models/helpers/CustomerConfigValidatorTest.java

```diff
@@ -96,6 +96,33 @@ public void testValidateDataContent_Storage_TwoParamsToCheck(
     assertEquals(expectedResult, result.size() == 0);
   }
 
+  @Test
```

Comment:
As the both next tests are almost the same, could we use one test with parameters?

Reply:
Yes. Done. Thanks for pointing out.